src/Build/Resources/Strings.resx

```diff
@@ -1714,13 +1714,19 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
       request a target to build itself (perhaps via a chain of other targets)
     </comment>
   </data>
+  <data name="CacheMissesNotAllowedInIsolatedGraphBuilds" UESanitized="false" Visibility="Public">
+    <value>MSB4252: Failed isolated build constraint: References should be built before projects referencing them. Project "{0}" has a reference to "{1}" ({2} target(s)) but the reference has not been previously built. In graph based builds this probably means that the reference is not explicitly specified as a project reference in "{0}".</value>
```

Comment:
`project reference` -> `ProjectReference`, and add a loc note to not translate it.